internal/core/cppapi_data.cpp

```diff
@@ -203,7 +204,7 @@ void fence(
             if (!axis.inrange(coordinate[voxel])) {
                 if (fillValue == nullptr) {
                     const std::string coordinate_str =
-                        "(" +std::to_string(x) + "," + std::to_string(y) + ")";
+                        "(" +utils::to_string_with_precision(x) + "," + utils::to_string_with_precision(y) + ")";
```

Comment:
This one is user-supplied data. Should we limit it as well? We will kinda limit it anyway by default 6 positions, but I am not sure how close the actual data is to each other - users generate the path themselves, likely in CDP coordinate system. Could it be that they usually differ at position 4 and 5? Idk...

From the other side I am not sure how this error message really helps users...

Reply:
The precision level required varies depending on the seismic survey. I am unsure of the appropriate precision level needed for the error message. However, I can change it to 6 significant digits if that is acceptable.

Reply:
I am not the one to know if that's acceptable. In a perfect world one could just briefly discuss that with our PO as he should have experience with what is usual for surveys.
But what what I know 6 should be good enough here.
So go get ready for merge!